lms/static/js/spec/student_account/access_spec.js

```diff
@@ -66,7 +66,9 @@
                             platform_name: 'edX',
                             login_form_desc: FORM_DESCRIPTION,
                             registration_form_desc: FORM_DESCRIPTION,
-                            password_reset_form_desc: FORM_DESCRIPTION
+                            password_reset_form_desc: FORM_DESCRIPTION,
+                            create_account_option: (typeof createAccountOption === 'undefined' ? true :
```

Comment:
ajaxSpyAndInitialize() is called in other tests as well, so in cases where createAccountOption argument is not passed, default true should be assigned